### DISCUSSION TITLE
Welcome to ActiPASS Discussions!

### DISCUSSION THREAD

**Discussion by pasan-h** [Ergo-Tools/ActiPASS#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions related to processing accelerometer data with ActiPASS, ActiPASS diary, generating variables etc..
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we  build together 💪.

**Comment by claaslendt** [comment]
Great idea!
